apps/router-e2e/__e2e__/server/app/api/externals+api.ts

```diff
@@ -0,0 +1,9 @@
+import { ExpoResponse } from 'expo-router/server';
+import fs from 'node:fs';
+import path from 'path';
+
+export function GET(): ExpoResponse {
+  console.log(fs);
+  // const txt = fs.readFileSync(path.join(__dirname, '../../../file.txt'), 'utf8');
+  return new ExpoResponse(path.join('a', 'b', 'c'), { status: 200 });
```

Comment:
This probably should have the `fs.readFileSync` and file, right? Maybe even just try to read the app name, and return it as app name?